openpype/hosts/maya/plugins/load/load_audio.py

```diff
@@ -80,8 +78,12 @@ def update(self, container, representation):
         asset = get_asset_by_id(
             project_name, subset["parent"], fields=["parent"]
         )
-        audio_node.sourceStart.set(1 - asset["data"]["frameStart"])
-        audio_node.sourceEnd.set(asset["data"]["frameEnd"])
+
+        source_start = 1 - asset["data"]["frameStart"]
```

Comment:
out of scope as well but docstring why `1 - frameStart`?

Reply:
I literally have no idea - this was the original logic written by @tokejepsen so maybe he knows more.

Reply:
I honestly dont remember why this is there. I'm up for removing it and fix it later if needed.

Reply:
For what it's worth, it was added here: https://github.com/ynput/OpenPype/pull/544

I'll mark this conversation resolved since it's not new logic introduced in this PR. I'd rather preserve what it did so that we can change the logic in a dedicated PR for it if needed.